moto/ec2/models.py

```diff
@@ -5562,6 +5562,30 @@ def modify_spot_fleet_request(
         return True
 
 
+class SpotPrice(object):
+    def describe_spot_price_history(self, instance_types=[], filters=None):
```

Comment:
Parameters should always default to None, ideally, instead of an empty list. It shouldn't impact the logic in this case